_episodes/20-crop-raster-data.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Crop raster data with rioxarray and geopandas"
+teaching: 40
+exercises: 20
+questions:
+- "How can I crop my raster data to the area of interest?"
+objectives:
+- "Crop raster data with a bounding box."
+- "Crop raster data with a polygon."
+- "Crop raster data within a geometry buffer."
+keypoints:
+- "Use `clip_box` in `DataArray.rio` to crop raster with a bounding box."
+- "Use `clip` in `DataArray.rio` to crop raster with a given polygon."
+- "Use `grow` in `GeoDataFrame` to make a buffer polygon of a (multi)point or a polyline. This polygon can be used to crop data."
```

Comment:
Lesson doesn't use `grow`. Should the last point be `buffer` from `geopandas`?

Reply:
```suggestion
- "Use `buffer` in `geopandas` to make a buffer polygon of a (multi)point or a polyline. This polygon can be used to crop data."
```
```suggestion
- "Use `buffer` in `geopandas` to make a buffer polygon of a (multi)point or a polyline. This polygon can be used to crop data."
```

Reply:
```suggestion
- "Use `buffer` in `geopandas` to make a buffer polygon of a (multi)point or a polyline. This polygon can be used to crop data."
```